kyo-core/js/src/main/scala/kyo/KyoAppBasePlatformSpecific.scala

```diff
@@ -0,0 +1,42 @@
+package kyo
+
+abstract class KyoAppBasePlatformSpecific extends KyoApp.Base[Throwable, Async & Resource]:
+    override def timeout: Duration = 10.minutes
+
+    protected def handle[A: Flat](v: A < (Async & Resource & Abort[Throwable]))(using Frame): A < (Async & Abort[Throwable]) =
+        Resource.run(v)
+
+    final override protected def args: Array[String] = _args
+
+    private var _args: Array[String] = null
+
+    private var concatenatedAsync: Any < (Async & Abort[Throwable]) = scala.compiletime.uninitialized
+    private var initCode: () => Unit                                = scala.compiletime.uninitialized
+
+    private def keepAlive(using Frame): Unit < (Async & Abort[Throwable]) = Async.sleep(timeout).map(_ => keepAlive)
+
+    final def main(args: Array[String]) =
+        this._args = args
+        if initCode ne null then initCode()
```

Comment:
I think we could throw an exception in case the `App` has no `run` calls. I imagine some users might try to define `def run` like in other effect systems and it'd just not execute anything silently.